src/main/java/org/kohsuke/github/GitHub.java

```diff
@@ -128,7 +129,13 @@
         } else {
             if (password!=null) {
                 String authorization = (login + ':' + password);
-                encodedAuthorization = "Basic "+new String(Base64.encodeBase64(authorization.getBytes()));
+                final Charset charset;
+                try {
+                    charset = Charset.forName("UTF-8");
```

Comment:
It's not a Jenkins plugin. Theoretically we could add Guava to github-api and then shade it. But it may be an overkill